packages/library/src/commonMain/kotlin/io/realm/Realm.kt

```diff
@@ -107,14 +117,25 @@ class Realm private constructor(configuration: RealmConfiguration, dbPointer: Na
      * Open a Realm instance. This instance grants access to an underlying Realm file defined by
      * the provided [RealmConfiguration].
      *
-     * FIXME Figure out how to describe the constructor better
-     * FIXME Once the implementation of this class moves to the frozen architecture
-     *  this constructor should be the primary way to open Realms (as you only need
-     *  to do it once pr. app).
+     * @param configuration The RealmConfiguration used to open the Realm.
      */
+    // FIXME Figure out how to describe the constructor better
     public constructor(configuration: RealmConfiguration) :
         this(configuration, RealmInterop.realm_open(configuration.nativeConfig))
 
+    /**
+     * Returns the results of querying for all objects of a specific type.
+     *
+     * The result is reflecting the state of the Realm at the invocation time, thus the results
+     * will not change on updates to the Realm and can be accessed from any thread.
```

Comment:
```suggestion
     * The result reflects the state of the realm at invocation time, so the results
     * do not change when the realm updates. You can access these results from any thread.
```
Passive -> active voice